docs/pages/versions/unversioned/config/metro.mdx

```diff
@@ -330,6 +330,25 @@ export default function Page() {
 }
 ```
 
+## Extending the Babel transformer
+
+Expo's Metro config uses a custom `transformer.babelTransformerPath` value to ensure `expo-babel-preset` is always used and web/Node.js environments are supported.
+
+If you want to extend the Babel transformer, ensure you import the upstream transformer from `@expo/metro-config/babel-transformer` instead of `metro-react-native-babel-transformer`, for example:
```

Comment:
```suggestion
Expo's Metro config uses a custom `transformer.babelTransformerPath` value to ensure `expo-babel-preset` is always used and web/Node.js environments are supported.

If you want to extend the Babel transformer, import the upstream transformer from `@expo/metro-config/babel-transformer` instead of `metro-react-native-babel-transformer`. For example:
```

We can be a bit more authoritative and concise at the same time.